incubator/java-microprofile/image/project/pom.xml

```diff
@@ -14,7 +14,7 @@
 
     <groupId>dev.appsody</groupId>
     <artifactId>java-microprofile</artifactId>
-    <version>0.2.21</version>
+    <version>{{.stack.semver.major}}.{{.stack.semver.minor}}.{{.stack.semver.patch}}</version>
```

Comment:
maybe since we're winding down we should just leave this and not require a new appsody CLI version?